### ISSUE TITLE
Unable to build nix package

### ISSUE THREAD

**Issue by L0ric0** [flyx/DSA-4.1-Heldendokument#16]
Trying to build the nix package fails with the following error:
```
error: builder for '/nix/store/c7wqyyl07y5qaifwcz689blqalj762qb-468.drv' failed with exit code 1;
       last 9 log lines:                                                                                            
       >
       > trying http://www.ulisses-spiele.de/download/468/
       >   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
       >                                  Dload  Upload   Total   Spent    Left  Speed
       > 100   322  100   322    0     0   6618      0 --:--:-- --:--:-- --:--:--  6708
       >   0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
       >   0     0    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0
       > curl: (22) The requested URL returned error: 404
       > error: cannot download 468 from any mirror
       For full logs, run 'nix log /nix/store/c7wqyyl07y5qaifwcz689blqalj762qb-468.drv'.
error: 1 dependencies of derivation '/nix/store/m2fl4c0aqpvcka2p5gw8pd0rxjf80r81-dsa41held.drv' failed to build
```

It seams Ulisses removed the file `http://www.ulisses-spiele.de/download/468/` is it possible to get is somewhere else?

**Comment by flyx** [comment]
This URL seems to host the same pdf: https://dsa-satinavsketten.de/fileadmin/downloads/offiziell/Regelmaterial&Errata/WdS-Handouts.pdf

Hash is different but extracting the background wallpaper seems to work. I don't have a completely working environment right now to test everything.

Since Ulisses removed the PDF from their site and the alternative URL is also owned by Ulisses, I wouldn't bet on it being available much longer, possibly this is only an oversight since that website seems to be abandoned. Since I am not allowed to redistribute the PDF myself, possibly the best I can do is to make this PDF dependency a non-flake input to the flake, so that people can inject their own local copy of the PDF.

**Comment by flyx** [comment]
Fixed in 09f662c07f8f2ee77a500ce1955815c9bbbfc71b. Also fixes another download that vanished.

Both files are now flake inputs, so they can potentially be overridden when importing the flake should they vanish. I suggest at least keeping a copy of the files.

Updated Flake needs recent Nix which is able to depend on arbitrary URLs. `2.8.0` is too old, `2.17.0` is working, not sure which is the minimum version that works.
